### DISCUSSION TITLE
Node-Red integration not loaded error message

### DISCUSSION THREAD

**Discussion by Peter200798** [zachowj/hass-node-red#326] [Q&A]
Hello everybody,

i have a problem with the node red integration.

After updating some intergrations and ha core, node red doesn't work anymore (for 6 days)

I checked the log of ha core and i found a warning from the node red integration (see attached files). 
I searched for the command with studio code server and replaced it, but it doesn't work.

I tried everything: delete and restart the node red integration, downgrading versions, running node red with the add on and external but nothing works and now i have no idea what to do.

Does anyone have an idea what i can do? Thanks in advanced.

![Bildschirmfoto_6-12-2024_22414_127 0 0 1](https://github.com/user-attachments/assets/92032060-65f4-4a83-8367-fb19a78b835b)
![Bildschirmfoto_6-12-2024_22028_homeassistant local](https://github.com/user-attachments/assets/b5d678f5-ec37-4ed9-a0e8-77f1209e1ba2)

**Comment by zachowj** [comment]
What is triggering the nodes that appear in the debug panel? If you're trying to send data through an entity node before Home Assistant has fully loaded the Node-RED integration, these errors will show up.

**Comment by Peter200798** [comment]
Hey,
thanks for the quick response.
These are sensors from "node-red-contrib-homeassistant-websocket" (number and binary) and they all are triggered by an inject node with a cycle time of 200ms. To understand: i read out modbus data and i created functions with flow.set/get to read out different areas from modbus. 
I didn't change the flows in node red and it worked before i do some updates. I don't know why.

You mean that the cycle time injects are to fast and loading of home assistant needs longer?

**Reply by zachowj** [reply]
> You mean that the cycle time injects are to fast and loading of home assistant needs longer?

Yes.

If I understand your use case correctly, this can be safely ignored. You can use a "catch" node to suppress these messages from the logs.

**Comment by Peter200798** [comment]
Okay i understand. 
I add a catch node to suppress the error messages, now these error messages are showen. Are these the result of the catch node?
Other ha nodes (actions) have the state connected but don't update these state.
It can be work also if i delay once at the beginning of the cycle times right?

Sorry for my questions, i'm new at home assistant <-> node-red 

Thank you very much for your support, i appreciate that :)

![Bildschirmfoto_7-12-2024_22758_127 0 0 1](https://github.com/user-attachments/assets/942a6cfc-3ed7-4a70-9fc9-f2eee2d51705)

**Reply by zachowj** [reply]
What do your Node-RED logs show? Is it repeatedly disconnecting and reconnecting?

**Reply by Peter200798** [reply]
<img width="950" alt="Screenshot 2024-12-07 222247" src="https://github.com/user-attachments/assets/dac228d0-41e3-4670-b721-9e4b0a0cb5ac">

 This is the log of the node-red companion integration.

**Reply by zachowj** [reply]
That's the Home Assistant logs. If you're using Node-RED as a Home Assistant add-on it would be under the add-on sections to find the logs.

**Reply by Peter200798** [reply]
I deleted the add on because of these problems, at the moment i use node red external.
